app/Models/SizeVariant.php

```diff
@@ -149,15 +152,15 @@ public function sym_links(): HasManyBidirectionally
 	public function getUrlAttribute(): string
 	{
 		if (
-			(AccessControl::is_admin() && Configs::get_value('SL_for_admin', '0') === '0') ||
-			Configs::get_value('SL_enable', '0') == '0'
+			!AccessControl::is_admin() && Configs::getValueAsBool('SL_for_admin', false) ||
+			!Configs::getValueAsBool('SL_enable', false)
```

Comment:
I assume the first negation is wrong. It should be

```suggestion
			AccessControl::is_admin() && !Configs::getValueAsBool('SL_for_admin', false) ||
			!Configs::getValueAsBool('SL_enable', false)
```